RSDKUtils.podspec

```diff
@@ -14,7 +14,8 @@ Pod::Spec.new do |s|
     'CLANG_MODULES_AUTOLINK'                                => 'YES',
     'CLANG_ALLOW_NON_MODULAR_INCLUDES_IN_FRAMEWORK_MODULES' => 'YES',
     'GCC_C_LANGUAGE_STANDARD'                               => 'gnu11',
-    'OTHER_CFLAGS'                                          => "'-DRPT_SDK_VERSION=#{s.version.to_s}'"
+    'OTHER_CFLAGS'                                          => "'-DRPT_SDK_VERSION=#{s.version.to_s}'",
+    'FRAMEWORK_SEARCH_PATHS'                                => '"${PODS_CONFIGURATION_BUILD_DIR}/Nimble"'
```

Comment:
Why do we need to link to test frameworks in the podspec?

Reply:
Because we provide extensions for them. Actually there's a problem because recently we proved that canImport doesn't work as expected in external modules. This means we have to make all dependencies explicit. For pod version we can just make a subspec for XCTests and Nimble (There's also a way to use objc to check availability of those deps in runtime). The question is what should we do about SPM version? create multiple products for the same package? I guess that's an equivalent of cocoapods subspecs

Reply:
Yep, please separate into separate subspecs and products